src/Location.cpp

```diff
@@ -1 +1,45 @@
-#include "Location.h"
\ No newline at end of file
+#include "Location.h"
+
+Location::Location(const char& letter, const int& num) : m_letter(letter), m_num(num) {}

```

Comment:
Pass char argument by value: Since char is a small primitive type, it is usually more efficient to pass it by value rather than by reference. This avoids the overhead of creating and dereferencing a reference object.